packages/services/src/index.ts

```diff
@@ -1 +1,7 @@
 export { default as SheetList } from './sheet/components/sheet-list/index';
+export { default as Section } from './sheet/components/sheet/section/index';
+export { default as PetDetails } from './sheet/components/sheet/pet-profile/index';
+export { default as UserProfile } from './sheet/components/sheet/user-profile/index';
+export { default as AddInput } from './sheet/components/sheet/add-input/index';
+export { default as AcceptButton } from './sheet/components/sheet/button/index';
+export { default as DatePick } from './sheet/components/sheet/date-pick/index';
```

Comment:
경로 맨 뒤에 있는 `index`는 빼도 됩니다!
```suggestion
export { default as SheetList } from './sheet/components/sheet-list';
```